app/wp-content/themes/kstg/page-vehicle.php

```diff
@@ -32,22 +32,22 @@
       <div class="carousel-images">
         <ul>
           <?php if( !empty($image1) ): ?>
-            <li><img src="<?php echo $image1['url']; ?>" alt="<?php echo $image1['alt']; ?>" width="200px" height="100px" /></li>
+            <li><img id="img1" src="<?php echo $image1['url']; ?>" alt="<?php echo $image1['alt']; ?>" width="200px" height="100px" /></li>
```

Comment:
Consider making this `id` something like 'img1-thumbnail' to differentiate it from the big images.